saleor/plugins/webhook/plugin.py

```diff
@@ -393,3 +402,67 @@ def process_payment(
             previous_value,
             **kwargs,
         )
+
+    def __send_shipping_webhook_sync(
+        self,
+        event_type: str,
+        previous_value: List[ExcludedShippingMethod],
+        payload: str,
+    ) -> List[ExcludedShippingMethod]:
+        excluded_methods_map = defaultdict(list)
+
+        # Gather responses from webhooks
+        webhooks = _get_webhooks_for_event(event_type)
+        for webhook in webhooks:
+            response_data = send_webhook_request_sync(
+                webhook.target_url, webhook.secret_key, event_type, payload
+            )
+            if response_data:
+                for method in parse_excluded_shipping_methods_response(response_data):
+                    excluded_methods_map[method.id].append(method)
```

Comment:
Maybe `excluded_methods_map[method.id].extend(parse_excluded_shipping_methods_response(response_data))`

Reply:
`excluded_methods_map` is a `defaultdict`, so I don't think `extend` is the way to go here. We can meddle with constructing a new dict here, using union or unpacking, but I don't think it's necessary.